### PR TITLE
BREAKING CHANGE: change ckb decimal to 18

### PR DESCRIPTION
**Pull request by zeroqn**

### CODE REVIEW

**Comment by TheWaWaR** [comment]
How about rollback the code format for next Pull Request ?

**Comment by zeroqn** [comment]
> How about rollback the code format for next Pull Request ?

I will revert it in this pr. I remember i have a commit for format only.

File: c/polyjuice_utils.h [hunk 1]
```diff
@@ -246,6 +245,9 @@ int parse_u64(const uint8_t data_be[32], uint64_t *value) {
 int parse_u128(const uint8_t data_be[32], uint128_t *value) {
   return parse_integer(data_be, (uint8_t *)value, sizeof(uint128_t));
 }
+int parse_u256(const uint8_t data_be[32], uint256_t *value) {
+  return parse_integer(data_be, (uint8_t *)value, sizeof(uint256_t));
+}
```
Comment (TheWaWaR) [inline c/polyjuice_utils.h:250]: Please update the comment of `parse_integer`.

File: c/polyjuice.h [hunk 2]
```diff
@@ -1368,17 +1364,21 @@ int run_polyjuice() {
 
   gw_reg_addr_t sender_addr = new_reg_addr(msg.sender.bytes);
 
-  ret = sudt_pay_fee(&context,
-                     g_sudt_id, /* g_sudt_id must already exists */
-                     sender_addr, fee);
+  uint8_t tmp_u256_be[32];
+  put_u128(fee, (uint8_t*)&tmp_u256_be);
+  uint256_t fee_u256 = {0};
+  ret = parse_u256(tmp_u256_be, &fee_u256);
```
Comment (TheWaWaR) [inline c/polyjuice.h:1370]: Just `memcpy((uint8_t *)(&fee_u256), (uint8_t*)(&fee), 16)` is ok, not necessary convert it to big endian first.

**Review by TheWaWaR — APPROVED** [review]

File: devtools/ci/integration-test.sh [hunk 3]
```diff
@@ -16,7 +16,7 @@ else
 fi
 cd $GODWOKEN_DIR
 # checkout https://github.com/nervosnetwork/godwoken/pull/659/head
-git fetch origin 0dd1500455ca81c23addadff05399925921aa352
+git fetch origin 8a8e0b7cd038fab41d44e6e4b936276d22c21171
```
Comment (Flouse) [inline devtools/ci/integration-test.sh]:
```suggestion
git fetch origin pull/675/merge
```

File: devtools/ci/integration-test.sh [hunk 4]
```diff
@@ -16,7 +16,7 @@ else
 fi
 cd $GODWOKEN_DIR
 # checkout https://github.com/nervosnetwork/godwoken/pull/659/head
```
Comment (Flouse) [inline devtools/ci/integration-test.sh:18]:
```suggestion
# checkout https://github.com/nervosnetwork/godwoken/pull/675/merge
```

**Review by Flouse — APPROVED** [review]

**Review by magicalne — APPROVED** [review]

**Pull request merged**